twin4build/api/codes/ml_layer/simulator_api.py

```diff
@@ -204,14 +195,19 @@ async def run_simulation_for_ventilation(self,input_dict: dict):
             startTime = datetime.datetime.strptime(input_dict["metadata"]["start_time"], self.time_format)
             endTime = datetime.datetime.strptime(input_dict["metadata"]["end_time"], self.time_format)
 
-            stepSize = int(self.config['model']['stepsize'])
+            #stepSize = int(self.config['model']['stepsize'])
```

Comment:
@avneet006 This is the hardcoded value

Reply:
No problem we will provide stepsize in input dictionary and from there we can get this value.